packages/CollabCodeEdit-Tests.package/CCEServerClassTest.class/instance/testRestart.st

```diff
@@ -0,0 +1,16 @@
+tests
+testRestart
+	| server previousServer |
+	server := nil.
+	previousServer := nil.
+
+	3 timesRepeat: [
```

Comment:
Blocks should be opened and closed in the same line.

Reply:
After a brief discussion, the style guide has been updated in favor of the above formatting.